docs/IndicatorSize.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Do we need a separate docs page just for indicator size? Can we not include it in the same documentation as `ActivityIndicator`?